lib/Backends/CPU/LLVMIRGen.cpp

```diff
@@ -1301,8 +1305,10 @@ void LLVMIRGen::generateLLVMIRForInstr(llvm::IRBuilder<> &builder,
     ShapeVector eDestDims = eBatchDims;
     eDestDims[BR->getAxis()] = 1;
 
-    auto *batchDims = emitConstArray(builder, eBatchDims);
-    auto *destDims = emitConstArray(builder, eDestDims);
+    auto *batchDims =
+        emitConstSizeTArray(builder, llvm::makeArrayRef(eBatchDims));
```

Comment:
Why these specific uses need to have an explicit makeArrayRef?
(I am not saying it is wrong, just curious :))